tools/sanity_checks.py

```diff
@@ -383,6 +383,9 @@ def check_new_release(self, name: str, builddir: str = '_build', deps=None, prog
                     if 'not found' in error:
                         print('cannot verify in wrapdb due to missing dependency')
                         return
+                elif 'ERROR: Could not execute Vala compiler: valac' in error:
+                    print('cannot verify in wrapdb due to missing dependency')
+                    return
```

Comment:
TBH, I'm frustrated that it still try to build on platforms we explicitly say won't work. @eli-schwartz forced adding that but I disagree, we just end up adding all kind of reasons here and waste time on it every time a new reason appears. Anyway... I guess that's fine...